external/vcm/vcm/data_transform.py

```diff
@@ -250,6 +251,37 @@ def implied_downward_radiative_flux_at_surface(
     return ds
 
 
+@register(
+    [
+        "Qm",
+        DELP,
+        DLW_SFC,
+        DSW_SFC,
+        DSW_TOA,
+        ULW_SFC,
+        ULW_TOA,
+        USW_SFC,
+        USW_TOA,
+        LHF,
+        SHF,
+    ],
+    ["implied_downward_radiative_flux_at_surface"],
+)
+def implied_downward_radiative_flux_at_surface_no_fine_res_t_nudging(ds, rectify=True):
+    """Assuming <Qm> = SHF + LHF + R_net + <T_nudge>."""
+    toa_net_flux = ds[DSW_TOA] - ds[USW_TOA] - ds[ULW_TOA]
+    surface_upward_flux = ds[LHF] + ds[SHF] + ds[USW_SFC] + ds[ULW_SFC]
+    surface_downward_flux = _tendency_to_implied_surface_downward_flux(
+        ds["Qm"], toa_net_flux, surface_upward_flux, ds[DELP], dim="z", rectify=rectify
```

Comment:
func takes many args...maybe shouldn't be a function. Would at least refactor any vertical averaging out of there.

Reply:
no longer adding this.